src/lib/components/tooltip/Tooltip.component.tsx

```diff
@@ -70,6 +70,8 @@ const TooltipText = styled.div`
   ul,
   ol {
     padding-inline-start: ${spacing.r16};
+    margin-block-start: 0;
```

Comment:
I think we also needed to change:

`padding: ${spacing.r4} ${spacing.r8};` to `padding: ${spacing.r8} ${spacing.r4};`

Reply:
Base on Valamir latest design, the padding-top is 4px, padding-bottom is 8px. So I think it was correct to keep the original value.  We had the confusion of Top is bigger than the Bottom is because there is the `margin-block-start: 10px;`  so by setting 0 to `margin-block-start` should be enough.